devito/ir/equations/algorithms.py

```diff
@@ -99,18 +102,30 @@ def lower_exprs(expressions, subs=None, **kwargs):
     --------
     f(x - 2*h_x, y) -> f[xi + 2, yi + 4]  (assuming halo_size=4)
     """
-    return _lower_exprs(expressions, subs or {})
+    return _lower_exprs(expressions, subs or {}, **kwargs)
 
 
-def _lower_exprs(expressions, subs):
+def _lower_exprs(expressions, subs, **kwargs):
+    # FIXME: Not sure why you can get this, but not access with index
+    sregistry = kwargs.get('sregistry')
```

Comment:
From below looks like it's mandatory not optional so need a valid default one not None

Reply:
the following is OK:

```
def _lower_exprs(..., sregistry=None, **kwargs):
    <use sregistry>
```